src/breadcrumbs/breadcrumbs.scss

```diff
@@ -50,9 +50,25 @@
     }
   }
 
+  > span:not([class]) {
```

Comment:
and what if it's `div` instead of `span`?

Reply:
I've replaced this line with `> *:not([class]) {`.

Reply:
And what if it has a class for some other purpose? You can just use `> *` and even remove these styles from the anchor overrides.

![image](https://user-images.githubusercontent.com/9084735/166492406-b7eaeae6-861c-4c0a-bb58-267f0e2bf023.png)

Reply:
Damn this PR is kicking my butt. 😣 Thank you for the example, I've implemented it almost exactly as you have it.  I did have to add `.iui-button { &.iui-button { @include iui-focus; } }` because the outline was appearing as 2px instead of 1px.

I did notice that `> * {` within `@mixin iui-breadcrumbs-item-overrides` is overriding `@mixin iui-breadcrumbs-text`.  Should be ok for pre CSS 1.0 I guess.